### ISSUE TITLE
[BUG] Chapter 7 (07_ensemble_learning_and_random_forests) in Jupyter Notebook

### ISSUE THREAD

**Issue by DaveBhatt** [ageron/handson-ml3#117]
**Describe the bug**

In 07_ensemble_learning_and_random_forests notebook

Exercise Solutions
6. If your AdaBoost ensemble underfits the training data, you can try increasing the number of estimators or reducing the regularization hyperparameters of the base estimator. You may also try slightly decreasing the learning rate.
7. If your Gradient Boosting ensemble overfits the training set, you should try increasing the learning rate. You could also use early stopping to find the right number of predictors (you probably have too many).

**Expected behavior**

To reduce underfitting i think we must try increasing the learning rate (not decreasing) and vice versa.
Pls correct me if i'm wrong.

**Screenshots**

![Screenshot 2023-12-26 122931](https://github.com/ageron/handson-ml3/assets/106701385/ef1b65f0-d20f-43ea-8266-4ecee3624efe)

**Comment by ageron** [comment]
Thanks for your question. Good catch, that's a mistake, these words should be reversed: decreasing the learning rate means that each estimator gets less weight, so the model is less likely to overfit. I'll fix that now.
